OWNERS

```diff
@@ -1,15 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
 approvers:
-- jsafrane
-- lpabon
-- msau42
-- saad-ali
-- xing-yang
+- kubernetes-csi-approvers
```

Comment:
Add lpabon as emeritus_approvers

Reply:
He is listed as that in release-tools/KUBERNETES_CSI_OWNERS_ALIASES.

Does it make sense to repeat that here?

Reply:
Furthermore, it raises the question of whether we want emeritus approvers or reviewers to be listed in *all* repos when removing them from the active lists in release-tools/KUBERNETES_CSI_OWNERS_ALIASES.

If we don't do that and list lpabon here, we become inconsistent. if we do it, then it defeats the purpose of rolling out changes automatically because we have to hand-edit all OWNERS files again.

Reply:
We don't need to list them in two places.  I forgot lpabon is already in the alias file.